### PR TITLE
Add Sapiens 1.0.0 (ready for review)

### PR DESCRIPTION
**Pull request by prihoda**
Add Sapiens 1.0.0 from PyPI: https://pypi.org/project/sapiens/

### CODE REVIEW

**Comment by prihoda** [comment]
This is ready to review. Windows is not supported due to missing pytorch dependency.

File: recipes/sapiens/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 41a8242da8ced4c4390ab6167950c6abb53ce84e3a7519282ccd680254d3119a
+
+build:
```
Comment (synapticarbors) [inline recipes/sapiens/meta.yaml:12]: It looks like this project has an `entry_point` for a console script. It should be added. See:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

and then a simple test of the command line tool should be added to the test section:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

File: recipes/sapiens/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License
```
Comment (synapticarbors) [inline recipes/sapiens/LICENSE:1]: I've opened a PR upstream to add the license file to the source distribution. If/when that's merged and there is a new upstream release of `sapiens`, this manually added copy should be removed.

**Review by synapticarbors — CHANGES_REQUESTED** [review]

File: recipes/sapiens/meta.yaml (hunk 1 above)
Comment (prihoda) [reply recipes/sapiens/meta.yaml:12]: Oh right, I created the entrypoint in setup.py, but it doesn't actually exist. So I deleted it from setup.py for now.

File: recipes/sapiens/LICENSE (hunk 2 above)
Comment (prihoda) [reply recipes/sapiens/LICENSE:1]: Merged and released 1.0.1, shall we change it here already? 

The new sha256 is `49eedaec6213410423ec3b46e2655fbf8731caee7de69fa2873a59497274bae7`

Comment (synapticarbors) [reply recipes/sapiens/LICENSE:1]: If you want you can update the version and sha256 here and delete the manually packaged license file

**Comment by synapticarbors** [comment]
conda-forge dropped using the `defaults` channel back in late-September 2021. There is a note in https://conda-forge.org/docs/user/announcements.html#announcements that says, "You can get the previous behaviour by using the `channel_sources` setting in `conda-forge.yml`." I don't see that field currently, so maybe it's been replaced by:
https://conda-forge.org/docs/maintainer/conda_forge_yml.html#channels

That said, I think it's ok to ignore the windows failure since this is a `noarch` package. I think it will be up to the end user to specify that they want to use the `defaults` channel when installing `sapiens`.

**Comment by prihoda** [comment]
> I think it will be up to the end user to specify that they want to use the defaults channel when installing sapiens.

Makes sense.

**Comment by prihoda** [comment]
@synapticarbors Let's merge this version, so that we include release 1.0.0. I can remove the LICENSE later.

**Review by synapticarbors — APPROVED** [review]

**Pull request merged**